src/core/Akka.Cluster.Tests.MultiNode/TransitionSpec.cs

```diff
@@ -87,32 +87,38 @@ private ImmutableHashSet<Address> MemberAddresses()
 
         private ImmutableHashSet<RoleName> Members()
         {
-            return MemberAddresses().Select(c => RoleName(c)).ToImmutableHashSet();
+            return MemberAddresses().Select(RoleName).ToImmutableHashSet();
         }
 
         private ImmutableHashSet<RoleName> SeenLatestGossip()
         {
-            // TODO: is it proper conversion of clusterView.seenBy flatMap roleName?
-            return ClusterView.SeenBy.Select(c => RoleName(c)).ToImmutableHashSet();
+            return ClusterView.SeenBy.Select(RoleName).ToImmutableHashSet();
         }
 
         private void AwaitSeen(params Address[] addresses)
         {
-            SeenLatestGossip().Select(c => GetAddress(c)).Should().BeEquivalentTo(addresses.ToImmutableHashSet());
+            AwaitAssert(() =>
```

Comment:
All of these methods needed to have `AwaitAssert` used inside of them in order to pass.